# Translations template for heat.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the heat project.
#
# Translators:
# Rodrigo Felix de Almeida <rodrigofelixdealmeida@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Heat\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-12-18 06:04+0000\n"
"PO-Revision-Date: 2014-12-17 02:38+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/projects/p/heat/"
"language/pt_BR/)\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: heat/api/aws/ec2token.py:161
msgid "Ec2Token authorization failed, no auth_uri specified in config file"
msgstr ""
"Autorização Ec2Token falhou, auth_uri não especificada no arquivo de "
"configuração"

#: heat/api/aws/utils.py:98
#, python-format
msgid "Request does not contain %s parameter!"
msgstr "Requisição não contém parâmetro %s!"

#: heat/api/cloudwatch/watch.py:253
msgid "Request does not contain required MetricData"
msgstr "Requisição não contém MetricData requerida"

#: heat/common/exception.py:116
msgid "Exception in string format operation"
msgstr "Exceção na operação de formato de sequência"

#: heat/common/heat_keystoneclient.py:135
msgid "Admin client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:156
msgid "Domain admin client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:186
msgid "Unknown version in auth_token_info"
msgstr "Versão desconhecida em auth_token_info"

#: heat/common/heat_keystoneclient.py:197
msgid "Keystone v3 API connection failed, no password trust or auth_token!"
msgstr ""

#: heat/common/heat_keystoneclient.py:209
msgid "Keystone client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:217
msgid "trust token re-scoping failed!"
msgstr ""

#: heat/common/heat_keystoneclient.py:221
msgid "Trust impersonation failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:364
#: heat/common/heat_keystoneclient.py:451
#, python-format
msgid "Failed to add user %(user)s to role %(role)s, check role exists!"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:44
msgid "failed to get lifecycle plug point classes"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:51
#, python-format
msgid "failed to instantiate stack lifecycle class %s"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:56
msgid "failed to sort lifecycle plug point classes"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:109
#, python-format
msgid "%(opname) %(ci)s failed for %(a)s on %(sid)s"
msgstr ""

#: heat/common/plugin_loader.py:99
#, python-format
msgid "Failed to import module %s"
msgstr "Falha ao importar módulo %s"

#: heat/common/wsgi.py:266
msgid "SIGTERM received"
msgstr "SIGTERM recebido"

#: heat/common/wsgi.py:275
msgid "SIGHUP received"
msgstr "SIGHUP recebido"

#: heat/common/wsgi.py:303
#, python-format
msgid "Removing dead child %s"
msgstr "Removendo filho inativo %s"

#: heat/common/wsgi.py:639
#, python-format
msgid "Exception handling resource: %s"
msgstr "Exceção ao manipular recurso: %s"

#: heat/common/wsgi.py:661
#, python-format
msgid "Returning %(code)s to user: %(explanation)s"
msgstr "Retornando %(code)s para o usuário: %(explanation)s"

#: heat/common/wsgi.py:735
#, python-format
msgid "Unexpected error occurred serving API: %s"
msgstr ""

#: heat/engine/api.py:41
msgid "Timeout conversion failed"
msgstr ""

#: heat/engine/api.py:298
msgid "Unexpected number of keys in watch_data.data!"
msgstr ""

#: heat/engine/environment.py:470
#, python-format
msgid "Failed to read %s"
msgstr ""

#: heat/engine/environment.py:482
#, python-format
msgid "Failed to parse %(file_path)s"
msgstr ""

#: heat/engine/environment.py:486
#, python-format
msgid "Failed to read %(file_path)s"
msgstr ""

#: heat/engine/plugin_manager.py:97
#, python-format
msgid "Failed to load %(mapping_name)s from %(module)s"
msgstr ""

#: heat/engine/plugin_manager.py:104
#, python-format
msgid "Invalid type for %(mapping_name)s from %(module)s"
msgstr ""

#: heat/engine/resource.py:481
msgid "Error marking resource as failed"
msgstr ""

#: heat/engine/resource.py:934 heat/engine/resource.py:961
#, python-format
msgid "DB error %s"
msgstr ""

#: heat/engine/resource.py:1085
#, python-format
msgid "signal %(name)s : %(msg)s"
msgstr ""

#: heat/engine/service.py:260
#, python-format
msgid "Unable to retrieve stack %s for periodic task"
msgstr ""

#: heat/engine/service.py:1389
msgid "Filtering by namespace/metric not yet supported"
msgstr ""

#: heat/engine/stack.py:728
#, python-format
msgid "Unexpected action %s passed to update!"
msgstr ""

#: heat/engine/stack.py:946
#, python-format
msgid "Unexpected action %s passed to delete!"
msgstr ""

#: heat/engine/stack.py:1130
#, python-format
msgid "Resource %(name)s delete failed: %(ex)s"
msgstr ""

#: heat/engine/stack.py:1139
#, python-format
msgid "Resource %(name)s create failed: %(ex)s"
msgstr ""

#: heat/engine/resources/eip.py:114
msgid ""
"No default floating IP pool configured. Set 'default_floating_pool' in nova."
"conf."
msgstr ""

#: heat/engine/resources/eip.py:330 heat/engine/resources/eip.py:353
msgid "Port not specified."
msgstr ""

#: heat/engine/resources/nova_floatingip.py:77
msgid ""
"Could not allocate floating IP. Probably there is no default floating IP "
"pool is configured."
msgstr ""

#: heat/engine/resources/remote_stack.py:149
#, python-format
msgid "exception: %s"
msgstr ""

#: heat/engine/resources/wait_condition.py:83
#, python-format
msgid "Metadata failed validation for %s"
msgstr ""

#: heat/engine/resources/aws/autoscaling_group.py:293
msgid "Failed sending error notification"
msgstr ""

#: heat/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "em uma chamada de laço de duração fixa"

#: heat/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "em chamada de laço dinâmico"

#: heat/openstack/common/policy.py:563 heat/openstack/common/policy.py:843
#, python-format
msgid "Failed to understand rule %s"
msgstr "Falha ao interpretar regra %s"

#: heat/openstack/common/policy.py:573
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Nenhum manipulador para correspondências do tipo %s"

#: heat/openstack/common/service.py:269
msgid "Unhandled exception"
msgstr "Exceção não tratada"
